manifests/app_stack.pp

```diff
@@ -13,8 +13,11 @@
 # @param [Integer] hdp_query_port
 #   Port to access HDP query service
 #
-# @param [Integer] hdp_ui_port
-#   Port to access HDP UI
+# @param [Integer] hdp_ui_http_port
+#   Port to access HDP UI via http
+#
+# @param [Integer] hdp_ui_https_port
+#   Port to access HDP UI via https
```

Comment:
We should add a tiny note that this port will be bound to the container, but will reject connections, if ui_tls is false.

Reply:
Agreed! FWIW, that tests this:

https://github.com/awrenn/hdp/blob/bb4b597485d5e67c7d5e44195583ae30e09ad1f0/manifests/app_stack.pp#L381-L387